build/resources/src/main/java/org/arquillian/warp/ftest/installation/ContainerInstaller.java

```diff
@@ -118,6 +122,9 @@ public void uninstallContainer(@Observes UninstallContainer event) {
 
             if (containerHome.exists()) {
                 FileUtils.deleteQuietly(containerHome);
+                if (FileUtils.deleteQuietly(containerHome) == false) {
```

Comment:
delete is called twice. Should remove line 124?

Reply:
D'oh, yes, copy/paste error. Thanks for spotting this. Fixed.

The WildFly 8.2.1 container does not seem to stop properly and thus will only be deleted partially. So I added the additional output, but I don't have an idea how to resolve it. See e.g. [https://github.com/ingwarsw/arquillian-suite-extension/issues/27](https://github.com/ingwarsw/arquillian-suite-extension/issues/27) for a similar problem.